controllers/positions.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs')
+const Positions = require('../data/positions.json')
+
+/** Description. takes params: idClient, jobDescription, vacancy, professionalProfiles and status. ID is generated automatically*/
```

Comment:
Use comments only when you think it is necessary to give context to understand the code. If the comment is only describing what the code does, it is not necessary since it is redundant.